frontend/src/index.css

```
@@ -116,11 +110,11 @@
   }
   body {
     @apply text-foreground;
-    background-color: var(--background);
+    background-color: var(--gradient-top);
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)